#motor pwm values (0 - 255)
int32 pwmFL
int32 pwmFR
int32 pwmRL
int32 pwmRR